src/main/java/io/kestra/plugin/prefect/cloud/CreateFlowRun.java

```diff
@@ -0,0 +1,240 @@
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Create a flow run in Prefect Cloud",
+    description = "Trigger a deployment and optionally wait for the flow run to complete."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Trigger a Prefect Cloud deployment and wait for completion",
+            full = true,
+            code = """
+                id: prefect_trigger
+                namespace: company.team
+                
+                tasks:
+                  - id: trigger_prefect_run
+                    type: io.kestra.plugin.prefect.cloud.CreateFlowRun
+                    accountId: "{{ secret('PREFECT_ACCOUNT_ID') }}"
+                    workspaceId: "{{ secret('PREFECT_WORKSPACE_ID') }}"
+                    deploymentId: "{{ secret('PREFECT_DEPLOYMENT_ID') }}"
+                    apiKey: "{{ secret('PREFECT_API_KEY') }}"
+                    wait: true
+                    pollFrequency: PT10S
+                """
+        ),
+        @Example(
+            title = "Trigger a Prefect Cloud deployment without waiting",
+            code = """
+                id: trigger_prefect_run
+                type: io.kestra.plugin.prefect.cloud.CreateFlowRun
+                accountId: "{{ secret('PREFECT_ACCOUNT_ID') }}"
+                workspaceId: "{{ secret('PREFECT_WORKSPACE_ID') }}"
+                deploymentId: "{{ secret('PREFECT_DEPLOYMENT_ID') }}"
+                apiKey: "{{ secret('PREFECT_API_KEY') }}"
+                wait: false
+                """
+        )
+    }
+)
+public class CreateFlowRun extends Task implements RunnableTask<CreateFlowRun.Output> {
+    private static final ObjectMapper OBJECT_MAPPER = JacksonMapper.ofJson();
+    
+    @Schema(
+        title = "Prefect Cloud API key",
+        description = "API key for authenticating with Prefect Cloud. You can create an API key from your Prefect Cloud account settings."
+    )
+    @NotNull
+    private Property<String> apiKey;
+
+    @Schema(
+        title = "Prefect Cloud account ID",
+        description = "The account ID (UUID) in Prefect Cloud."
+    )
+    @NotNull
+    private Property<String> accountId;
+
+    @Schema(
+        title = "Prefect Cloud workspace ID",
+        description = "The workspace ID (UUID) in Prefect Cloud."
+    )
+    @NotNull
+    private Property<String> workspaceId;
+
+    @Schema(
+        title = "Deployment ID",
+        description = "The deployment ID (UUID) to create a flow run from."
+    )
+    @NotNull
+    private Property<String> deploymentId;
+
+    @Schema(
+        title = "Wait for flow run completion",
+        description = "Whether to wait for the flow run to complete before continuing. If true, the task will poll the flow run status until it reaches a terminal state (COMPLETED, FAILED, CANCELLED, etc.)."
+    )
+    @Builder.Default
+    private Property<Boolean> wait = Property.of(true);
```

Comment:
`Property.ofValue` instead of `Property.of` everywhere (if it's a value)

```suggestion
    private Property<Boolean> wait = Property.ofValue(true);
```